src/FrameRipper.py

```diff
@@ -0,0 +1,79 @@
+import cv2
+import numpy as np
+import os 
+
+#Frame Ripper Script WIP
+#Working on adding subset functionality in order to randomly grab sets of frames instead of all of them
+#
+#Steve
+
+#Function to scan the video files, processes and adds them to Array video_list
+def getVideoFiles():
```

Comment:
```suggestion
def get_video_files():
```

Reply:
Also maybe add a (defaulted) parameter for the video directory.
```suggestion
def get_video_files(video_path = 'Videos'):
```

Reply:
Also consider methods & variables name casing throughout.